config/redirects.yml

```diff
@@ -395,6 +396,17 @@ redirects:
   "/blog/tag/career-change-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
   "/blog/tag/changing-career-to-teaching": "/life-as-a-teacher/teaching-as-a-career/change-careers"
   "/blog/tag/interviews": "/how-to-apply-for-teacher-training/teacher-training-interview"
+  "/blog/tag/applications": "/how-to-apply-for-teacher-training/teacher-training-application"
```

Comment:
would be good to order **all** of these tag redirects either alphabetically or maybe by destination?